src/proto/mgmt/control.proto

```diff
@@ -45,4 +45,6 @@ service MgmtCtl {
     rpc FetchFioConfigPaths(EmptyReq) returns(stream FilePath) {};
     // List features supported on remote storage server/DAOS system
     rpc ListFeatures(EmptyReq) returns(stream Feature) {};
+    // Query raw SPDK NVMe SSD device health stats
+    rpc DeviceHealthQuery(QueryHealthReq) returns (QueryHealthResp) {};
```

Comment:
NvmeHealth instead?

Reply:
I decided to remove this definition, and just reuse StorageScan